config/application.yml.example

```diff
@@ -9,6 +9,8 @@
 # Figaro requires all values to be explicit strings.
 
 email_from: 'no-reply@login.gov'
+idv_max_attempts: '3'
+idv_attempt_window: '24'
```

Comment:
What do you think about adding `in_hours` to make it clearer what the number represents?